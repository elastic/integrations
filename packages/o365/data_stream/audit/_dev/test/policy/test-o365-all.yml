input: o365audit
data_stream:
  vars:
    application_id: application-id
    client_secret: client-secret
    key: /tmp/path_to/key
    key_passphrase: key-passphrase
    tenants:
      - tenant-id
    tenant_names:
      - "tenant-id: domain-name"
    content_type:
      - "Audit.AzureActiveDirectory"
      - "Audit.Exchange"
      - "Audit.SharePoint"
      - "Audit.General"
      - "DLP.All"
    api: |
      - setting:
          example: true
    tags:
      - forwarded
      - o365-audit
    preserve_original_event: true
    processors: |-
      - add_fields:
          target: foo
          fields:
            bar: baz
