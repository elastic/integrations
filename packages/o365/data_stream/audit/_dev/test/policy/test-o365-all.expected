inputs:
    - data_stream:
        namespace: ep
      meta:
        package:
            name: o365
      name: test-o365-all-o365
      streams:
        - api:
            - setting:
                example: true
          api.preserve_original_event: true
          application_id: application-id
          client_secret: ${SECRET_0}
          content_type:
            - Audit.AzureActiveDirectory
            - Audit.Exchange
            - Audit.SharePoint
            - Audit.General
            - DLP.All
          data_stream:
            dataset: o365.audit
            type: logs
          key: /tmp/path_to/key
          key_passphrase: ${SECRET_1}
          processors:
            - add_fields:
                fields:
                    tenants:
                        tenant-id: domain-name
                target: _conf
            - add_fields:
                fields:
                    bar: baz
                target: foo
          publisher_pipeline.disable_host: true
          tags:
            - forwarded
            - o365-audit
            - preserve_original_event
          tenant_id:
            - tenant-id
      type: o365audit
      use_output: default
output_permissions:
    default:
        _elastic_agent_checks:
            cluster:
                - monitor
        _elastic_agent_monitoring:
            indices: []
        uuid-for-permissions-on-related-indices:
            indices:
                - names:
                    - logs-o365.audit-ep
                  privileges:
                    - auto_configure
                    - create_doc
secret_references:
    - {}
    - {}
